test/jdk/java/awt/Mixing/AWT_Mixing/MixingPanelsResizing.java

```diff
@@ -75,17 +78,21 @@ public class MixingPanelsResizing {
     private static int borderShift;
 
     private static int frameBorderCounter() {
+
         String JAVA_HOME = System.getProperty("java.home");
         try {
-            Process p = Runtime.getRuntime().exec(JAVA_HOME + "/bin/java FrameBorderCounter");
+            Process p = Runtime.getRuntime()
+                    .exec(JAVA_HOME + "/bin/java FrameBorderCounter");
```

Comment:
Did I already asked this question?

Why can't the code in `FrameBorderCounter` be run in the same JVM? I'm pretty sure it can be.

It's like for a separate refactoring, yet I'd like to have a tracking bug for getting rid of launching another process to count something that can be done by executing `FrameBorderCounter.main` directly and ensuring the frame is disposed of.